packages/app-store/server.ts

```diff
@@ -79,26 +80,24 @@ export async function getLocationGroupedOptions(
   const integrations = await getEnabledApps(credentials, true);
 
   integrations.forEach((app) => {
+    // All apps that are labeled as a locationOption are video apps.
     if (app.locationOption) {
       // All apps that are labeled as a locationOption are video apps. Extract the secondary category if available
-      let category =
+      let groupByCategory =
         app.categories.length >= 2
-          ? app.categories.find(
-              (category) =>
-                !([AppCategories.video, AppCategories.conferencing] as string[]).includes(category)
-            )
-          : app.category;
-      if (!category) category = AppCategories.conferencing;
+          ? app.categories.find((groupByCategory) => !defaultVideoAppCategories.includes(groupByCategory))
+          : app.categories[0] || app.category;
```

Comment:
`category` is deprecated, let's use `categories[0]` and fallback to category for some edgecases. We should plan to remove it after scanning all the apps.